src/content/docs/2/recipe/notifications.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Notifications
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+In this recipe we'll take a look at sending a native notification to your user. We'll do this by using the official notification plugin.
+
+## Setup
+
+First you have to install the notification plugin in your backend.
+
+<Tabs>
+    <TabItem label="Cargo.toml">
+    ```toml
+    [dependencies]
+    tauri-plugin-notification = "2.0.0-alpha"
+    ```
+    </TabItem>
+    <TabItem label="lib.rs">
+    ```rust
+    #[cfg_attr(mobile, tauri::mobile_entry_point)]
+    pub fn run() {
+        tauri::Builder::default()
+            // Initialize the plugin
+            .plugin(tauri_plugin_notification::init())
+            .run(tauri::generate_context!())
+            .expect("error while running tauri application");
+    }
+    ```
+    </TabItem>
+</Tabs>
+
+## Usage
+
+Once you're done with the general setup you're free to use the notification plugin in either your frontend or straight from the backend.
+
+### Frontend
```

Comment:
I think I like the phrasing of "frontend" and "backend" and that distinction. Could you do a quick write-up (maybe add it to the CONTRIBUTING guide in this PR) that explains what we mean by that to make it easy to any contributors?

Reply:
I can also see this would be a very good place to write a concept of "Frontend vs. Backend" and link to it. Maybe we could add a `TODO:` here and slot that into here in the future?

Reply:
Sounds good to me :)